tests/Controller/OrderApiTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Tests\Controller;
+
+use Lakion\ApiTestCase\JsonApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @author Daniel Gorgan <danut007ro@gmail.com>
+ */
+class OrderApiTest extends JsonApiTestCase
+{
+    /** @var array */
+    private static $authorizedHeaderWithContentType = [
+        'HTTP_Authorization' => 'Bearer SampleTokenNjZkNjY2MDEwMTAzMDkxMGE0OTlhYzU3NzYyMTE0ZGQ3ODcyMDAwM2EwMDZjNDI5NDlhMDdlMQ',
+        'CONTENT_TYPE' => 'application/json',
+    ];
+
+    public function testAccessDeniedException()
+    {
+        $this->client->request('GET', '/api/orders/');
+
+        $response = $this->client->getResponse();
+        $this->assertResponse($response, 'authentication/access_denied_response', Response::HTTP_UNAUTHORIZED);
+    }
+
+    public function testRequiredData()
+    {
+        $this->loadFixturesFromFile('authentication/api_administrator.yml');
+        $this->loadFixturesFromFile('resources/orders.yml');
+
+        $this->client->request('POST', '/api/orders/', [], [], static::$authorizedHeaderWithContentType);
+
+        $response = $this->client->getResponse();
+        $this->assertResponse($response, 'order/create_validation_fail_response', Response::HTTP_BAD_REQUEST);
+    }
+
+    public function testCreateOrder()
+    {
+        $this->loadFixturesFromFile('authentication/api_administrator.yml');
+        $channelData = $this->loadFixturesFromFile('resources/orders.yml');
+
+        $data =
+<<<EOT
+{
+    "channel": {$channelData['channel-web']->getId()},
+    "currencyCode": "{$channelData['currency']->getCode()}"
+}
```

Comment:
Wrong indent, brackets should be at the same level as the rest of function body.